docs/user_guide/mitigation.rst

```diff
@@ -74,6 +74,81 @@ Preprocessing
    
 .. currentmodule:: fairlearn.preprocessing
 
+Preprocessing algorithms transform the dataset to mitigate possible unfairness
+present in the data.
+Preprocessing algorithms in Fairlearn follow the `scikit-learn transformers
+<https://scikit-learn.org/stable/modules/generated/sklearn.base.TransformerMixin.html#sklearn.base.TransformerMixin>`_
+class, meaning that they can :code:`fit` to the dataset and :code:`transform` it
+(or :code:`fit_transform` to fit and transform in one go).
+
+.. _correlation_remover:
+
+Correlation Remover
+~~~~~~~~~~~~~~~~~~~
+Sensitive features can be correlated with non-sensitive features in the dataset.
+By applying the :code:`CorrelationRemover`, these correlations are filtered out while
+details from the original data are retained as much as possible (as measured by the
+least-squares error). The user can control the level of filtering via the :code:`alpha`
+parameter. The removal of correlation is mathematically described as:
+
+.. math::
+        \min _{\mathbf{z}_{1}, \ldots, \mathbf{z}_{n}} \sum_{i=1}^{n}\left\|\mathbf{z}_{i}
+        -\mathbf{x}_{i}\right\|^{2} \\
+        \text{subject to} \\
+        \frac{1}{n} \sum_{i=1}^{n} \mathbf{z}_{i}\left(\mathbf{s}_{i}-\overline{\mathbf{s}}
+        \right)^{T}=\mathbf{0}
+
+The amount of filtering (as controlled with the :code:`alpha` parameter) is then
+described as follows:
+
+.. math::
+        X_{\mathrm{tfm}}=\alpha X_{\text {filtered }}+(1-\alpha) X_{\mathrm{orig}}
+
+In the example below, a subset of the `adult dataset <https://www.openml.org/d/1590>`_
+is loaded and the correlation between sex and the non-sensitive features is removed.
+This procedure will drop the sensitive features from the dataset.
+
+.. doctest:: mitigation
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> from fairlearn.preprocessing import CorrelationRemover
+    >>> import pandas as pd
+    >>> from sklearn.datasets import fetch_openml
+    >>> data = fetch_openml(data_id=1590, as_frame=True)
+    >>> X = data.data[['age', 'fnlwgt', 'education-num', 'sex']]
+    >>> X = pd.get_dummies(X)
+    >>> X = X.drop('sex_Male', axis=1)
+    >>> cr = CorrelationRemover(sensitive_feature_ids=['sex_Female'])
+    >>> cr.fit(X)
+    CorrelationRemover(sensitive_feature_ids=['sex_Female'])
+    >>> X_transform = cr.transform(X)
+
+In order to see what the :code:`CorrelationRemover` has done,
```

Comment:
How about something like:

In order to see the effect of  :class:`CorrelationRemover`, we visualize how the correlation matrix has changed after the transformation of the dataset.